src/CoreWCF.Http/src/CoreWCF.Http.csproj

```diff
@@ -4,6 +4,17 @@
     <AssemblyName>CoreWCF.Http</AssemblyName>
     <PackageId>CoreWCF.Http</PackageId>
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+    <VersionPrefix>0.0.0</VersionPrefix>
+    <VersionSuffix>pre-beta</VersionSuffix>
+    <Owners>dotnetfoundation</Owners>
+    <Authors>mconnew, biroj, .NET Foundation and Contributors</Authors>
```

Comment:
This is going to be difficult to maintain. I don't like the idea that package metadata like this needing to be updated with each release. I did a very quick look at what other packages in the dotnet foundation do and the one which made most sense to me would be "CoreWCF Contributors". We'll make sure there's a contributors section in the release notes and we might even be able to create a stable URL for it to include here as you can create GitHub url's to content which is dynamic in that you can point to the latest version of a document of a specific branch. For now just use "CoreWCF contributors" and we'll try to make something which is more explicit before RTM.

Reply:
I was wrong, according to the documentation, the Authors field is about the nuget account which owns the package so that people can search for other packages from the same owner. From the [,nuget spec](https://docs.microsoft.com/en-us/nuget/reference/nuspec#authors):

> A comma-separated list of packages authors, matching the profile names on nuget.org. These are displayed in the NuGet Gallery on nuget.org and are used to cross-reference packages by the same authors.

So that field is yet to be worked out.

Reply:
How about for the time being let's keep ".NET Foundation and Contributors"

Reply:
As other packages use `%ProjectName% Contributors`, I'm fine with "CoreWCF Contributors" for now as the .NET Foundation didn't author anything.